### PR TITLE
Add pronunciation audio

### PR DESCRIPTION
**Pull request by mundanevision20**
Hi @radomirbosak 

As the title reads I'd like to add the option to display the urls for audio files provided by duden.de for the pronunciation of words. 

Looking forward to your feedback.

Regards, Henning

### CODE REVIEW

**Comment by radomirbosak** [comment]
Thank you for this pull request @mundanevision20 ! I really appreciate it.

Please give me a few days to find time for review. Thank you!

File: duden/display.py [hunk 1]
```diff
@@ -130,46 +130,49 @@ def describe_word(word):
     print(yellow("=" * len(word.title)))
 
     if word.phonetic:
-        print(white(_("Pronunciation:"), bold=True), word.phonetic)
```
Comment (radomirbosak) [inline duden/display.py:133]: Please do not remove the `_` function. It is needed for the localization using the `gettext` module.

For example users with Spanish localization see this:

```
LANG=es_ES.UTF-8 duden Kragen
Kragen, der
===========
Tipo de palabra: Substantiv, maskulin
Frecuencia: 2/5
Separación: Kra|gen
Origin: mittelhochdeutsch krage = Hals, Kehle, Nacken; Kragen (1), ursprünglich = Schlund
...
```
(i know the translation is incomplete)

File: duden/display.py [hunk 2]
```diff
@@ -130,46 +130,49 @@ def describe_word(word):
...
     if word.origin:
-        print(white(_("Origin:"), bold=True), word.origin)
+        print(white("Origin:", bold=True), word.origin)
 
     if word.grammar_overview:
-        print(white(_("Grammar:"), bold=True), word.grammar_overview)
+        print(white("Grammar:", bold=True), word.grammar_overview)
+
+    if word.word_pronunciation_audio:
+        print('{label} {content}'.format(label=white('Pronunciation audio:', bold=True), content=white(word.word_pronunciation_audio, bold=False)))
```
Comment (radomirbosak) [inline duden/display.py:165]: This line is too long. This project uses a maximum of 88 characters (as seen in configuration file:)
```
$ git grep 'line-length'
pyproject.toml:line-length = 88
```

An easiest way to fix this is to run the `make autoformat` Makefile task which autoformats the code into a standard form.

File: duden/display.py (hunk 1 above)
Comment (mundanevision20) [reply duden/display.py:133]: sorry my fault. I didn't knew that.

File: duden/display.py (hunk 2 above)
Comment (mundanevision20) [reply duden/display.py]: Understood. I fixed it.

**Comment by radomirbosak** [comment]
I pushed your changes to master branch. Thank you for contributing! :) 

The only change I made was to rename the function from `word_pronunciation_audio` to `pronunciation_audio_url`, because the word context is obvious and adding `_url` makes it clear that it is the url link to the sound, and not the sound data itself.

And sorry it taking so long!

**Pull request closed without merging**